### PR TITLE
Switch from term to termcolor 1.0 for prompt colors

### PR DESCRIPTION
**Pull request by ghost**
This switches the prompt color crate from term to termcolor.

### CODE REVIEW

File: src/prompt.rs [hunk 1]
```diff
@@ -201,58 +201,41 @@ impl Prompt {
...
             return safe_prompt();
         }
-        if write!(t, "carapace").is_err() {
+        if write!(&mut buffer, "carapace").is_err() {
             println!("Failed to write to term!");
         }
 
         if let Ok(cwd) = env::current_dir() {
-            if t.fg(term::color::BRIGHT_BLUE).is_err() {
+            if buffer.set_color(color.set_fg(Some(Color::Blue))).is_err() {
```
Comment (netromdk) [inline src/prompt.rs]: I would like to keep the `BRIGHT_BLUE` if possible.

File: src/prompt.rs [hunk 2]
```diff
@@ -201,58 +201,41 @@ impl Prompt {
...
-        }
-
-        // Get inner cursor (it was moved above) and read what was written to its data vector.
-        let mut inner_cursor = t.into_inner();
-        if inner_cursor.seek(SeekFrom::Start(0)).is_err() {
-            return safe_prompt();
-        }
-
-        let mut out = Vec::new();
-        if inner_cursor.read_to_end(&mut out).is_err() {
```
Comment (netromdk) [inline src/prompt.rs:251]: It's great we don't need this anymore! 👍

File: src/prompt.rs [hunk 3]
```diff
@@ -201,58 +201,41 @@ impl Prompt {
...
 
-        // In-memory cursor using a vector for data.
-        let cursor = Cursor::new(Vec::new());
-
-        let t = term::TerminfoTerminal::new(cursor);
-        if t.is_none() {
-            return safe_prompt();
-        }
-        let mut t = t.unwrap();
+        let bufwtr = BufferWriter::stderr(ColorChoice::Always);
```
Comment (netromdk) [inline src/prompt.rs:204]: What exactly does the `ColorChoice::Always` mean here?

**Review by netromdk — CHANGES_REQUESTED** [review]
This is awesome! Thanks. Only one change I'd like :)

File: src/prompt.rs (hunk 1 above)
Comment (ghost) [reply src/prompt.rs]: Give that a try. I switched to bright blue using an rgb value.

File: src/prompt.rs (hunk 3 above)
Comment (netromdk) [reply src/prompt.rs:204]: Ah:
> Try very hard to emit colors. This includes emitting ANSI colors on Windows if the console API is unavailable.

https://docs.rs/termcolor/1.0.5/termcolor/enum.ColorChoice.html#variant.Always

**Review by netromdk — APPROVED** [review]

**Pull request merged**